components/proofly/sources/new-notification-data-created/new-notification-data-created.mjs

```diff
@@ -0,0 +1,52 @@
+import common from "../common/polling.mjs";
+
+export default {
+  ...common,
+  key: "proofly-new-notification-data-created",
+  name: "New Notification Data Created",
+  description: "Emit new event instantly when notification data is received. [See the documentation](https://proofly.io/developers)",
```

Comment:
I'm not sure we should use the word "instantly" since it's a polling source. Wdyt?
```suggestion
  description: "Emit new event when notification data is received. [See the documentation](https://proofly.io/developers)",
```